Hardware/Hardware/Printer/AirPrintReceipt/AirPrintReceiptPrinterService.swift

```diff
@@ -13,12 +13,18 @@ public final class AirPrintReceiptPrinterService: PrinterService {
         return info
     }()
 
-    public init() { }
+    private var receiptContent: ReceiptContent?
+
+    override public init() {
```

Comment:
Overriding this initializer should not be needed, as far as I can tell, given it just calls super

Reply:
Fixed with 93e4cfadca2f28510be75effb3f5d2845e2578ad